Dockerfile.amd64

```diff
@@ -44,7 +44,9 @@ RUN dnf install -y 'dnf-command(config-manager)' && \
     # Need these to build runit.
     yum install -y wget glibc-static gcc && \
     # Ensure security updates are installed.
-    yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical
+    yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical && \
+    # Update systemd related packages due to CVEs (advisories RHSA-2021:2717 and RHSA-2021:1611)
+    yum -y update-minimal --advisory=RHSA-2021:2717 --advisory=RHSA-2021:1611
```

Comment:
I think the CVEs are showing up because we’re using old versions (8.3) of ubi and centos to build our node image. You can see this at the scan results for v3.20.0: https://quay.io/repository/calico/node/manifest/sha256:913955a36179a53d36a0df7f26319a5278991eea2514eab339cce15df24c220c?tab=vulnerabilities The fix versions for these two libs is `0:239-45.el8_4.2` -- note the `8_4` -- so I think if we just update ubi and centos from 8.3 -> 8.4 that should remove the clear up the vuln scan.

Reply:
We can verify whether the changes work by building an image from this branch and pushing it to quay.io

Reply:
thanks for catching this, @lmm! the packages were being downgraded further along and the advisories were still showing up on Quay. The correct way to do this is to update the ubi image and centos repos to 8.4, which I just pushed.